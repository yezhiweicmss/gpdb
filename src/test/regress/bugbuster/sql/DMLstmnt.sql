CREATE FUNCTION addlab1111(integer,integer) RETURNS integer
AS 'select $1 + $2;'
LANGUAGE SQL 
IMMUTABLE
CONTAINS SQL
RETURNS NULL ON NULL INPUT;

ALTER FUNCTION addlab1111(integer,integer) RENAME TO addition111;


DROP FUNCTION addition111(integer,integer);

CREATE USER jonathan11 WITH PASSWORD 'abc1';
CREATE USER jonathan12 WITH PASSWORD 'abc2';

ALTER USER jonathan11 RENAME TO jona11;
ALTER USER jonathan12 RENAME TO jona12;

DROP USER jona11;
DROP USER jona12;

CREATE USER jonathan12 WITH PASSWORD 'abc2';

ALTER USER jonathan11 RENAME TO jona11;
ALTER USER jonathan12 RENAME TO jona12;

CREATE GROUP marketing WITH USER jona11,jona12;

ALTER GROUP marketing RENAME TO market;

DROP GROUP market;
DROP USER jona11;
DROP USER jona12;

select unnest('{}'::text[]);
