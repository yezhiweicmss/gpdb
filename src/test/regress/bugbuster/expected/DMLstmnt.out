CREATE FUNCTION addlab1111(integer,integer) RETURNS integer
AS 'select $1 + $2;'
LANGUAGE SQL
IMMUTABLE
CONTAINS SQL
RETURNS NULL ON NULL INPUT;
ALTER FUNCTION addlab1111(integer,integer) RENAME TO addition111;
DROP FUNCTION addition111(integer,integer);
CREATE USER jonathan11 WITH PASSWORD 'abc1';
NOTICE:  resource queue required -- using default resource queue "pg_default"
CREATE USER jonathan12 WITH PASSWORD 'abc2';
NOTICE:  resource queue required -- using default resource queue "pg_default"
ALTER USER jonathan11 RENAME TO jona11;
NOTICE:  MD5 password cleared because of role rename
ALTER USER jonathan12 RENAME TO jona12;
NOTICE:  MD5 password cleared because of role rename
DROP USER jona11;
DROP USER jona12;
CREATE USER jonathan12 WITH PASSWORD 'abc2';
NOTICE:  resource queue required -- using default resource queue "pg_default"
ALTER USER jonathan11 RENAME TO jona11;
ERROR:  role "jonathan11" does not exist
ALTER USER jonathan12 RENAME TO jona12;
NOTICE:  MD5 password cleared because of role rename
CREATE GROUP marketing WITH USER jona11,jona12;
NOTICE:  resource queue required -- using default resource queue "pg_default"
ERROR:  role "jona11" does not exist
ALTER GROUP marketing RENAME TO market;
ERROR:  role "marketing" does not exist
DROP GROUP market;
ERROR:  role "market" does not exist
DROP USER jona11;
ERROR:  role "jona11" does not exist
DROP USER jona12;
select unnest('{}'::text[]);
 unnest 
--------
(0 rows)

